SnapSpire/Features/Discover/View/DiscoverView.swift

```diff
@@ -0,0 +1,35 @@
+import SwiftUI
+
+struct DiscoverView: View {
+    var body: some View {
+        NavigationView {
```

Comment:
We should not use NavigationView, we should use the new API

Reply:
👍 I understand you!